src/main/java/tech/ypsilon/bbbot/discord/command/VerifyCommand.java

```diff
@@ -0,0 +1,151 @@
+        mailSessionProperties.put("mail.smtp.starttls.enable", "true");
+        mailSessionProperties.put("mail.smtp.host", SettingsController.getValue("mail.smtp.host"));
+        mailSessionProperties.put("mail.smtp.port", SettingsController.getValue("mail.smtp.port"));
+        mailSessionProperties.put("mail.smtp.ssl.trust", SettingsController.getValue("mail.smtp.ssl.trust"));
+
+        mailSession = Session.getInstance(mailSessionProperties, new Authenticator() {
+            @Override
+            protected PasswordAuthentication getPasswordAuthentication() {
+                return new PasswordAuthentication(
+                        (String) SettingsController.getValue("mail.smtp.username"),
+                        (String) SettingsController.getValue("mail.smtp.password")
+                );
+            }
+        });
+    }
+
+    @Override
+    public String[] getAlias() {
+        return new String[]{"verify"};
+    }
+
+    @Override
+    public String getDescription() {
+        return "E-Mail Verifizierung um die Student-Rolle zu erhalten";
+    }
+
+    @Override
+    public void onExecute(GuildMessageReceivedEvent event, String[] args) {
+        event.getMessage().delete().queue();
+        event.getAuthor().openPrivateChannel().flatMap(privateChannel ->
+                privateChannel.sendMessage("For security reasons, the `verify` command "
+                        + "is only allowed per direct message")).queue();
+    }
+
+    @Override
+    public void onPrivateExecute(PrivateMessageReceivedEvent event, String[] args) {
+        if (args.length < 1) {
+            event.getChannel().sendMessage(HELP_EMBED).queue();
+        } else if (StudentUtil.isStudentCode(args[0])) {
+            // TODO: 1. check if user is already verified
+            // TODO: 2. check if user has already initiated, but not completed a verification sequence
+            // TODO: 3. check if u code is already verified under a different user
+            // TODO: 4. if none of the above apply, initiate verification sequence
+        } else {
+            // TODO: 1. check if user is already verified
+            // TODO: 2. check if user has already initiated, but not completed a verification sequence
+                // TODO 2.1: check if code is correct and verify the user (+insert into database)
+                // TODO 2.2: else send error
+            // TODO: 3. check if u code is already verified under a different user
+            // TODO: 4. if none of the above apply, send help message
+        }
+    }
+
+    private void sendEmail(String recipient) throws MessagingException {
+        Message message = new MimeMessage(this.mailSession);
+        message.setFrom(new InternetAddress((String) SettingsController.getValue("mail.smtp.address")));
+        message.setRecipients(Message.RecipientType.TO, InternetAddress.parse(recipient + "@student.kit.edu"));
+        message.setSubject("");
```

Comment:
Mail subject-header should be set, otherwise some mailservers (e.g. yahoo) won't accept the messages. 
(Exactly ONE header needs to be present, duplicate headers will also cause issues!)

Reply:
done: d63611d